CRM/Mailing/Form/Search.php

```diff
@@ -92,7 +92,7 @@ public function setDefaultValues() {
       $defaults['status_unscheduled'] = 1;
     }
     if ($parent->get('scheduled')) {
-      $statusVals = array('Scheduled', 'Complete', 'Running', 'Canceled');
+      $statusVals = array('Scheduled', 'Complete', 'Running', 'Paused', 'Canceled');
```

Comment:
So just to be clear -- this status already exists,  but it wasn't showing up in the search form?

The API seems to get a list of valid statuses by calling `CRM_Core_SelectValues::getMailingJobStatus()` -- perhaps the form should do the same?

Reply:
Ths status does exist, but it wasn't enabled on the page load. Updated the PR to use the above function to get the statuses on the Search form.